app/tests/forms/test_submission_integration.py

```diff
@@ -45,26 +46,28 @@ def _create_submission_data_with_text_answer(field, answer_text):
 
 
 @pytest.fixture()
-def form():
+def event_form():
```

Comment:
Uenig i dette egt. Vi bør ha alle tester på forms by default også heller lage ekstra tester til event_form der det er egen funksjonalitet

Reply:
Enig! Dette var egentlig bare en rename. Å fikse dette burde nok være eget issue. :stuck_out_tongue:

Reply:
Laget #330 for dette.